tensorflow/python/keras/losses.py

```diff
@@ -871,7 +872,11 @@ def sparse_categorical_crossentropy(y_true, y_pred, from_logits=False, axis=-1):
 
 @keras_export('keras.metrics.binary_crossentropy',
               'keras.losses.binary_crossentropy')
-def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):  # pylint: disable=missing-docstring
+def binary_crossentropy(y_true, y_pred, from_logits=False, label_smoothing=0):
+  """Computes the binary cross-entropy between `y_true` and `y_pred`.
+
+  `binary_crossentropy = - y_true*log(y_pred) - (1 - y_true)*log(1 - y_pred)`
```

Comment:
Missed this last time, can we change this to `loss =` for consistency with other losses.